website/cue/reference/components/sources/base/host_metrics.cue

```diff
@@ -72,7 +72,7 @@ base: components: sources: host_metrics: configuration: {
 			"""
 		required: false
 		type: array: {
-			default: ["cpu", "disk", "filesystem", "load", "host", "memory", "network", "cgroups"]
```

Comment:
I think you need to document the newly published metrics here: https://github.com/vectordotdev/vector/blob/e90df6b75ef9d27a0c7341da18fadba1787db1c7/website/cue/reference/components/sources/host_metrics.cue#L71-L259